classes/cluster/mk22_lab_basic/fuel/config.yml

```diff
@@ -3,28 +3,24 @@ classes:
 - system.linux.system.single
 - system.linux.system.repo.tcp_salt
 - system.openssh.client.lab
-- system.salt.master.pkg
+- system.salt.master.git
 - system.reclass.storage.salt
 - system.salt.minion.pki.authority
 - system.salt.minion.pki.certificate
 - system.sphinx.server.doc.reclass
-- system.keystone.client.single
-- system.keystone.client.service.ceilometer
-- system.keystone.client.service.nova21
-- system.mysql.client.single
```

Comment:
Same comment. Do you really to remove this class? And note that cluster/mk20_stacklight_basic/fuel/config.yml has system.keystone.client.service.nova.